fem/fe_coll.cpp

```diff
@@ -3468,16 +3469,18 @@ void NURBSFECollection::SetOrder(int Order) const
 
 NURBSFECollection::~NURBSFECollection()
 {
-   delete ParallelepipedFE;
-   delete QuadrilateralFE;
+   delete PointFE;
    delete SegmentFE;
+   delete QuadrilateralFE;
+   delete ParallelepipedFE;
 }
 
 const FiniteElement *
 NURBSFECollection::FiniteElementForGeometry(Geometry::Type GeomType) const
 {
    switch (GeomType)
    {
+      case Geometry::POINT:       return PointFE;
```

Comment:
Is `PointFE` ever used, or is it just added here for completeness?

Reply:
I think it is used as boundary of a 1D mesh. Not 100% sure though...

Reply:
Checked it and it is indeed necessary for the 1D BCs.